tests/tests_suite_test.go

```diff
@@ -178,14 +178,6 @@ func getRunningAAQ(aaqClient *clientset.Clientset) (*aaqv1.AAQ, error) {
 }
 
 func updateAAQNamespaceSelector(aaqClient *clientset.Clientset, k8sClient *kubernetes.Clientset, selector *metav1.LabelSelector) error {
-	// Ensure AAQ is deployed and overwrite default namespace selector to target any namespace
-	aaqWebhook, err := k8sClient.AdmissionregistrationV1().MutatingWebhookConfigurations().Get(context.Background(), cluster.MutatingWebhookConfigurationName, metav1.GetOptions{})
-	if err != nil {
-		return err
-	}
-
-	originalWebhookSelector := aaqWebhook.Webhooks[0].NamespaceSelector
-
```

Comment:
An idea for an optimization: what you can do is check if the webhook is already defined with the given selector, and if it is, return early without trying to change it.

Reply:
Done